cylc/flow/workflow_files.py

```diff
@@ -1695,17 +1695,17 @@ def install_workflow(
         if source_link.is_symlink():
             # Condition represents a broken symlink.
             raise WorkflowFilesError(
-                f'Workflow source dir is not accessible:'
-                f' "{source_link.resolve()}".\n'
-                f'Restore the source or modify the "{source_link}" symlink'
-                ' to continue.'
+                f'Workflow source, "{source_link}" is a broken '
+                f'symlink to "{source_link.resolve()}".'
```

Comment:
This one is tricky, I'm trying to think what the most useful message to present in terms of how the user might end up in this situation. I think the most likely case is they renamed the source dir? In which case fixing the problem will be fiddly, involving changing the source symlink target. We might want a CLI command to redirect a workflow's source symlink, e.g. `cylc update-source <workflow_name>`? If so, I will create an issue

Reply:
IMO Not a Cylc problem, doesn't need a Cylc solution:

```suggestion
                f'Symlink broken {source_link} -> {source_link.resolve()}.'
```